File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio34.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio34 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o seu peso: ");
        float peso = entrada.nextFloat();
        System.out.println("Insira a sua altura: ");
        float altura = entrada.nextFloat();

        float imc = peso / (altura * altura);
        if (imc < 18.5){
            System.out.println("Abaixo do peso");
        } else if (imc >= 18.5 && imc < 25) {
            System.out.println("Peso ideal");
        }else if (imc >= 25 && imc < 30) {
            System.out.println("Sobrepeso");
        }else if (imc >= 30 && imc < 40) {
            System.out.println("Obesidade");
        }else{
            System.out.println("Obesidade mórbida");
        }
    }
}

File: guanabara-python/src/lista03/exercicio34.py
peso = float(input("Insira o peso: "))
altura = float(input("Insira a altura: "))
imc = peso / (altura * altura)

if imc < 18.5:
    print(f"IMC: {imc}. Classificação: Abaixo do peso.")
elif imc >= 18.5 and imc < 25:
    print(f"IMC: {imc}. Classificação: Peso ideal.")
elif imc >= 25 and imc < 30:
    print(f"IMC: {imc}. Classificação: Sobrepeso.")
elif imc >= 30 and imc < 40:
    print(f"IMC: {imc}. Classificação: Obesidade.")
else:
    print(f"IMC: {imc}. Classificação: Obesidade mórbida.")
